src/datadog/datadog_agent_config.cpp

```diff
@@ -21,7 +21,7 @@ Expected<DatadogAgentConfig> load_datadog_agent_env_config() {
 
   if (auto raw_rc_poll_interval_value =
           lookup(environment::DD_REMOTE_CONFIG_POLL_INTERVAL_SECONDS)) {
-    auto res = parse_int(*raw_rc_poll_interval_value, 10);
+    auto res = parse_double(*raw_rc_poll_interval_value);
```

Comment:
Note to myself: a double is too big but I don't have time (don't want yet) to write (and test) a `parse_float` function.